EleksTubeHAX_pio/src/Mqtt_client_ips.cpp

```diff
@@ -11,21 +11,59 @@
  */
 
 #include "Mqtt_client_ips.h"
-#include "WiFi.h"         // for ESP32
+#include <WiFi.h>         // for ESP32
 #include <PubSubClient.h> // Download and install this library first from: https://www.arduinolibraries.info/libraries/pub-sub-client
 #include <ArduinoJson.h>
 #include "TempSensor.h"
 #include "TFTs.h"
 #include "Backlights.h"
 #include "Clock.h"
+#ifdef MQTT_USE_TLS
+#include <WiFiClientSecure.h> // for secure WiFi client
+
+WiFiClientSecure espClient;
+// ************ Let's encrypt CA Root Certificate  *****************
+// Valid To	04 Jun 2035
+const char *SSL_CA_PEM = "-----BEGIN CERTIFICATE-----\n"
```

Comment:
Is there anything to be said against not embedding the certificate in the code but reading it from the file system instead? Or at least include it from a header file here and leave it up to the user to decide which root CA is used.

Reply:
Sounds like a good idea to load the cert from a PEM file in the data partition. I will modify the code and test it.